docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+# Event Broadcast and Subscription
```

Comment:
@divyk Ok. So, please guide @kilig how to update the toc_all.md file.

Reply:
@kilig , since it is a new document can you please update in https://github.com/Samsung/tizen-docs/blob/master/docs/application/toc_all.md also.

Please make sure that it comes under .NET application > Guides > Application Management